src/documentation/ptransform-style-guide.md

```diff
@@ -0,0 +1,488 @@
+---
+layout: default
+title: "PTransform Style Guide"
+permalink: /documentation/ptransform-style-guide/
+---
+
+# PTransform style guide
+
+## A style guide for writers of new reusable PTransform's
+
+{:toc}
```

Comment:
I recall this having been a pain for the release guide, but it works there. https://raw.githubusercontent.com/apache/beam-site/asf-site/src/contribute/release-guide.md

Try nesting it inside a bullet?